docs/trino-connector/supported-catalog.md

```diff
@@ -122,6 +122,31 @@ call gravitino.system.alter_catalog(
 if you need more information about catalog, please refer to:
 [Create a Catalog](../manage-relational-metadata-using-gravitino.md#create-a-catalog).
 
+## Passing Trino connector configuration
+A Gravitino catalog is implemented by the Trino connector, so you can pass the Trino connector configuration to the Gravitino catalog.
+For example, you want to set the `hive.config.resources` configuration for the Hive catalog, you can pass the configuration to the
+Gravitino catalog like this:
+
+```sql
+call gravitino.system.create_catalog(
+    'gt_hive',
+    'hive',
+    map(
+        array['metastore.uris', 'trino.bypass.hive.config.resources'],
+        array['thrift://trino-ci-hive:9083', "/tmp/hive-site.xml,/tmp/core-site.xml"]
+    )
+);
+```
+
+The prefix `trino.bypass` is used to indicate Gravitino connector to pass the Trino connector configuration to the Gravitino catalog in Trino.
```

Comment:
Pleas try to polish this sentence.

Reply:
I don't understand what's wrong with this sentence, do you have any suggestions?